procrastinate/sql/migrations/delta_0.15.2_001_fix_procrastinate_defer_periodic_job.sql

```diff
@@ -0,0 +1,52 @@
+CREATE OR REPLACE FUNCTION procrastinate_defer_periodic_job(
```

Comment:
I think it's ok because we don't change the function signature but neithertheless, I'd be reassured if we could delete the function, as a rule. `CREATE OR REPLACE FUNCTION` doesn't work in some cases and even if I'm 90% we're not in one of those cases, I4d rather not take useless risks.

Reply:
Agree. I was thinking about the same.